static/.htaccess

```
@@ -14,6 +14,10 @@ RewriteRule ^optimize/3.14.0/(.*)$ /optimize/$1 [R=302,L]
 # ----------------------
 # None here yet, but please place them here when you have them.
 
+# Remove Connector update guides entirely
+RewriteRule ^docs/components/connectors/custom-built-connectors/update-guide/(.*)$ /docs/components/connectors/custom-built-connectors/connector-sdk/$1 [R=301,L]
+RewriteRule ^docs/next/components/connectors/custom-built-connectors/update-guide/(.*)$ /docs/next/components/connectors/custom-built-connectors/connector-sdk/$1 [R=301,L]
```

```suggestion
RewriteRule ^docs/components/connectors/custom-built-connectors/update-guide/(.*)$ /docs/components/connectors/custom-built-connectors/connector-sdk/ [R=301,L]
```

Discussed in person with @akeller: (1) all pages should go to the same target instead of preserving their individual path; (2) `next` isn't needed because the `current` one will already cover everything when 8.7 is cut.

---

`$1` get me every time.... thank you 🙏